Added unkown salvager shuttle event

jorgun opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
Frontier are you there?  Station this is NX- 432 Please Answer
## About the PR
<!-- What did you change in this PR? -->
added a new shuttle event. a salvager shuttle from frontier station has gottne lost and driften into unknown terretory
all credits for the ship to Frontier staton "https://frontierstation14.com/"

## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
eh


## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
added new shuttle event and salvager ghostroles
## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.
-->
![image](https://github.com/user-attachments/assets/56c0119d-77e2-4fa2-8474-98b28701bbfc)

## Requirements
<!-- 
Due to influx of PR's we require to ensure that PR's are following the correct guidelines.

Please take a moment to read these if its your first time.

Check the boxes below to confirm that you have in fact seen these (put an X in the brackets, like [X]):
-->
- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up. Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add: Added New Salvager Shuttle Event

whateverusername0 reviewed (approved):
expecting gang wars with this one

jorgun commented:
some notes:
removed the crusher wepons - salvage can find there own
added 1 pka
added salvager access to the airalarm and apc 
added a tank dispencer vendor
conteplated adding a salvage vending machine - still unsure about this and running out of space
smes shuid have power for a little less then 30 minutes and 1 stack of plasma shud fuel the plasma generator for another 30minutes.
